.github/labeler.yml

```diff
@@ -32,4 +32,12 @@ golang:
 
 python:
   - ./**/*.py
-  - src/idl_gen_python.cpp
\ No newline at end of file
+  - src/idl_gen_python.cpp
+
+java:
+    - ./**/*.java
```

Comment:
I'm not sure if the extra spaces matter, but we should keep it consistent at least.

Reply:
Sure, guess I need glasses 🤦